neuralop/training/tests/test_trainer.py

```diff
@@ -200,12 +176,18 @@ def test_incremental():
         out_channels=1,
     )
     
-    trainer = Trainer(model=model,
-                      n_epochs=20,
-                      callbacks=callbacks)
+    trainer = IncrementalFNOTrainer(
+        model=model,
+        n_epochs=20,
+        incremental_loss_gap=False,
+        incremental_grad=True,
+        incremental_grad_eps=0.9999,
+        incremental_buffer=5,
+        incremental_max_iter=1,
+        incremental_grad_max_iter=2,)
 
     optimizer = torch.optim.Adam(model.parameters(), 
-                                lr=8e-3, 
+                                lr=3e-4, 
```

Comment:
I see we changed the default LR - new one seems more reasonable, but can we add some documentation on this and the parameters of incremental FNO generally (either in the doctoring or user guide) @Robertboy18 ?

Reply:
Sure I can add that!